### PR TITLE
[NUI] Fix color format to #RRGGBBAA from #AARRGGBB

### PR DESCRIPTION
**Pull request by rabbitfor**
Signed-off-by: Jiyun Yang <ji.yang@samsung.com>

### CODE REVIEW

**Comment by rabbitfor** [comment]
This is related with https://github.com/Samsung/TizenFX/pull/3225

File: src/public/XamlBuild/CompiledConverters/ColorTypeConverter.cs [hunk 1]
```diff
@@ -196,24 +196,24 @@ static Color FromHex(string hex)
...
                 case 6: //#rrggbb => ffrrggbb
                     return FromRgb((int)(ToHex(hex[idx++]) << 4 | ToHex(hex[idx++])),
                             (int)(ToHex(hex[idx++]) << 4 | ToHex(hex[idx++])),
                             (int)(ToHex(hex[idx++]) << 4 | ToHex(hex[idx])));
 
-                case 8: //#aarrggbb
-                    var a1 = ToHex(hex[idx++]) << 4 | ToHex(hex[idx++]);
-                    return FromRgba((int)(ToHex(hex[idx++]) << 4 | ToHex(hex[idx++])),
+                case 8: //#rrggbbaa
+                    var red = ToHex(hex[idx++]) << 4 | ToHex(hex[idx++]);
```
Comment (bshsqa) [inline src/public/XamlBuild/CompiledConverters/ColorTypeConverter.cs]: Do we need to handle red value separately?

Comment (rabbitfor) [reply src/public/XamlBuild/CompiledConverters/ColorTypeConverter.cs]: Fixed!

**Review by bshsqa — APPROVED** [review]

**Review by AdunFang — COMMENTED** [review]
It is OK.

**Pull request merged**